pysteps/visualization/basemaps.py

```diff
@@ -11,13 +11,13 @@
     plot_geography
     plot_map_cartopy
 """
+from cartopy.mpl.geoaxes import GeoAxesSubplot

```

Comment:
As Cartopy is an optional dependency, this import should be included in a try ... except ImportError ... statement.